protocol/migrations/sqlite/1684979808_sync_activity_center_notifications.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE activity_center_states ADD COLUMN clock INT DEFAULT 0;
+ALTER TABLE activity_center_notifications ADD COLUMN clock INT DEFAULT 0;
```

Comment:
Do you think we should add an index or two to cover the new column? I haven't checked the SQL queries to see if any could be impacted, so I'm not sure the lack of an index can actually be a problem.

Reply:
I'll check it